siac/walletcmd.go

```diff
@@ -192,8 +192,11 @@ func walletinitcmd() {
 }
 
 // walletload033xcmd loads a v0.3.3.x wallet into the current wallet.
-func walletload033xcmd(source string) {
-	password, err := speakeasy.Ask("Wallet password: ")
+func walletload033xcmd() {
+	fmt.Print("Enter 033x Location: ")
+	var source string
+	fmt.Scanln(&source)
```

Comment:
I agree. More interactivity is generally a bad thing because it makes siac harder to script.